mmaction/core/bbox/assign_sampling.py

```diff
@@ -0,0 +1,23 @@
+import mmcv
+
+from . import assigners, samplers
+
+
+def build_assigner(cfg, **kwargs):
+    if isinstance(cfg, assigners.BaseAssigner):
+        return cfg
+    elif isinstance(cfg, dict):
+        return mmcv.runner.obj_from_dict(cfg, assigners, default_args=kwargs)
+    else:
+        raise TypeError('Invalid type {} for building a sampler'.format(
+            type(cfg)))
```

Comment:
MMAction2 will polish docs manually by pylint for convenience according to https://github.com/open-mmlab/mmpose/pull/283. There will be some warning like `R1705: Unnecessary "elif" after "return" (no-else-return)`, `R1720: Unnecessary "else" after "raise" (no-else-raise)` etc.

```python
def build_assigner(cfg, **kwargs):
    if isinstance(cfg, assigners.BaseAssigner):
        return cfg
    if isinstance(cfg, dict):
        return mmcv.runner.obj_from_dict(cfg, assigners, default_args=kwargs)
    raise TypeError(f'Invalid type {type(cfg)} for building a sampler')
```